assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -205,12 +240,14 @@ const Row = ({
   icon,
   text,
   rightText,
+  extraClasses,
 }: {
   icon?: ReactElement
   text: string | ReactElement
   rightText?: string
+  extraClasses?: string
 }) => (
-  <div className="m-minischedule__row">
+  <div className={`m-minischedule__row ${extraClasses ? extraClasses : ""}`}>
```

Comment:
we have a helper `className()` in `dom.ts` that will avoid adding the trailing whitespace. If it makes the call simpler, you could change it to take `(string | undefined)[]` instead of `string[]`.